odoo/cli/i18n.py

```diff
@@ -0,0 +1,225 @@
+            or path.suffixes[-1] not in IMPORT_EXTENSIONS
+        )}:
+            _logger.warning("Ignoring invalid paths: %s", ', '.join(invalid_paths))
+            paths -= invalid_paths
+        if not paths:
+            self.import_parser.error("No valid path was provided")
+
+        with Registry.new(parsed_args.db_name).cursor() as cr:
+            env = odoo.api.Environment(cr, odoo.SUPERUSER_ID, {})
+            translation_importer = odoo.tools.translate.TranslationImporter(cr)
+            language = self._get_languages(env, [parsed_args.language])
+            if not language:
+                self.import_parser.error("No valid language has been provided")
+            for path in paths:
+                with open(path, "rb") as infile:
+                    translation_importer.load(infile, path.suffixes[-1].removeprefix('.'), language.code)
+            translation_importer.save(overwrite=parsed_args.overwrite)
+
+    def _export(self, parsed_args):
+        if parsed_args.output:
+            path = Path(parsed_args.output)
+            extension = (path.suffixes and path.suffixes[-1]) or '-'
+            if extension not in EXPORT_EXTENSIONS + ['-']:
+                self.export_parser.error(
+                    f"Extensions allowed for --output are {', '.join(EXPORT_EXTENSIONS)}")
+            if len(parsed_args.languages) != 1:
+                self.export_parser.error(
+                    "When --output is specified, one single --language must be supplied")
+            if extension == '.csv':
+                # The default limit for CSV fields in the module is 128KiB, which is not
+                # quite sufficient to import images to store in attachment. 500MiB is a
+                # bit overkill, but better safe than sorry
+                csv.field_size_limit(500 << 20)
+
+        export_pot = 'pot' in parsed_args.languages
+        if export_pot:
+            if parsed_args.output and parsed_args.output.endswith('.csv'):
+                self.export_parser.error(
+                    "Cannot export template in .csv format, please specify a language.")
+            parsed_args.languages.remove('pot')
+
+        with Registry.new(parsed_args.db_name).cursor() as cr:
+            env = odoo.api.Environment(cr, odoo.SUPERUSER_ID, {})
+
+            # We want to log invalid parameters
+            modules = env['ir.module.module'].with_context(active_test=False).search_fetch(
+                [('name', 'in', parsed_args.modules)], ['name', 'state'])
+            if not_found_module_names := set(parsed_args.modules) - set(modules.mapped("name")):
+                _logger.warning("Ignoring not found modules: %s", ", ".join(not_found_module_names))
+            if not_installed_modules := modules.filtered(lambda x: x.state != 'installed'):
+                modules_str = ", ".join(not_installed_modules.mapped("name"))
+                _logger.warning("Ignoring not installed modules: %s", modules_str)
```

Comment:
In the first `if` you don't use a local variable, but you do in the second. Let's pick one and be consistent in the two cases.